front-end/package.json

```diff
@@ -11,9 +12,16 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.4.0",
     "bootstrap": "^5.3.0-alpha1",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
```

Comment:
You do not need `express` `mongodb` `mongoose` `morgan` in front-end package.json it should be in backend package.json, delete from here.